test/test_utils.go

```diff
@@ -239,7 +240,7 @@ func verifySignature(ctx context.Context, t *testing.T, c *clients, tr *v1beta1.
 	logger := logging.FromContext(ctx)
 
 	// Initialize the backend.
-	backends, err := chainsstrorage.InitializeBackends(ctx, c.PipelineClient, c.KubeClient, logger, tr, *cfg)
+	_, _, backends, err := chains.AbstractFactory(ctx, c.PipelineClient, taskrun.SecretPath, c.KubeClient, *cfg, logger)
```

Comment:
Perhaps we should keep the InitializeBackends func? No need to create all of the clients / signers / formatters if we only care about the backends.

Reply:
`InitializeBackends` function still exists, but the name was changed to unexported `allBackends(...)` to keep consistent and was moved to `factory.go`.
How about making `AllBackends` exported and just call `AllBackends` here instead of using `AbstractFactory` to initialize all?

Reply:
SGTM